troposphere/static/js/components/projects/resources/instance/details/sections/PlaybookStatus.jsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import Backbone from "backbone";
+import StatusLight from "components/projects/common/StatusLight";
+
+export default React.createClass({
+    displayName: "Status",
+
+    propTypes: {
+        instancePlaybookHistory: React.PropTypes.instanceOf(Backbone.Model).isRequired
+    },
+
+    isInError: function(instancePlaybookStatus) {
+        if(instancePlaybookStatus == 'error' || instancePlaybookStatus == 'deploy_error') {
+            return true;
+        }
+        return false;
+    },
+    renderProgressBar: function(percentComplete) {
+        if(percentComplete == 100) {
+            return;
+        }
+        var barStyle = {
+            width: "0%"
+        };
+
+        barStyle.width = percentComplete + "%";
+
+            return (<div className="progress">
+                <div className="progress-bar progress-bar-success progress-bar-striped active" style={barStyle} title={barStyle.width}>
```

Comment:
No worries if this is outside the scope or too tall for this use case. Just want to start pointing people to these components over bootstrap.